web/src/components/Dialogs/ImageDialog.tsx

```diff
@@ -0,0 +1,106 @@
+import { AnimatePresence, motion } from "framer-motion";
+import Image from "next/future/image";
+import { MdFavorite, MdFavoriteBorder, MdLibraryAdd } from "react-icons/md";
+
+import Button from "../Inputs/Button";
+import IconButton from "../Inputs/IconButton";
+import Card from "../Surfaces/Card";
+import Author from "../Widgets/Author";
+import LikesCounter from "../Widgets/LikesCounter";
+import StyledDialog from "./StyledDialog";
+
+interface Props {
+  src: string;
+  prompt: string;
+  likes: number;
+  isLiked: boolean;
+  onLikedChange: (isLiked: boolean) => void;
+  authorName: string;
+  authorAvatar: string;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export default function ImageDialog({
+  src,
+  prompt,
+  likes,
+  isLiked = false,
```

Comment:
Last thing - you also have `isLiked` with a default here and not in the `Props`. I suggest just removing the `= false` so the type isn't a lie.

Reply:
Wait didn't I update this? not sure what happened, I'll change it